### ISSUE TITLE
environs.json or genomes.json issue

### ISSUE THREAD

**Issue by gianfilippo** [scfurl/henipipe#7]
Hi,

I am trying to modify the environs.json or genomes.json files to get the pipeline to work on my cluster, but modifications have no effect.

Could you please explain how to modify the config files ?

Thanks

**Comment by scfurl** [comment]
Sorry you are having trouble.  I have only successfully implemented on PBS and SLURM.  So currently those are the only two I would feel comfortable supporting. If you wanted to try messing with it you can edit the SLURM block in the environs.json file with commands to suit your cluster.  SLURM is the default so you will want to edit that JSON block unless you change the --cluster flag…. Hope that helps and good luck

From: Gianfilippo Coppola ***@***.***>
Date: Tuesday, October 11, 2022 at 5:53 PM
To: scfurl/henipipe ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [scfurl/henipipe] environs.json or genomes.json issue (Issue #7)

Hi,

I am trying to modify the environs.json or genomes.json files to get the pipeline to work on my cluster, but modifications have no effect.

Could you please explain how to modify the config files ?

Thanks

**Comment by gianfilippo** [comment]
Hi,

Thanks for your prompt reply and suggestions!

I think I figured it out. My mistake was to try and modify the json files under the data DIR, not the ones 'installed'. Although I also tried to reinstall, after modifying the files and it did not work....probably I made some other mistake there.
Anyway, after I modified the correct files, it worked. 
I did have to add a line to your SLURM section for it to run on my cluster: #SBATCH -p general, where general is the queue ID. Probably not necessary for everyone.

I think it would be nice to be able to have custom config files that one can load at command line.

Best
